src/third-party/gutenberg.stories.mdx

```diff
@@ -0,0 +1,568 @@
+    {`<figure class="wp-block-image size-large is-style-default">
+        <img
+          src="https://s.w.org/images/core/5.3/MtBlanc1.jpg"
+          alt="Mont Blanc"
+          class="wp-image-130"
+          srcset="
+            https://s.w.org/images/core/5.3/MtBlanc1.jpg         500w,
+            https://s.w.org/images/core/5.3/MtBlanc1-300x225.jpg 300w
+          "
+          sizes="(max-width: 500px) 100vw, 500px"
+        />
+        <figcaption>Mont Blanc appears—still, snowy, and serene.</figcaption>
+      </figure>`}
+  </Story>
+</Preview>
+
+## Media-Text
+
+Combines media and textual content in a wrapper with a class of `wp-block-media-text`.
+The media content is contained in a wrapper with class `wp-block-media-text__media`
+and its text sibling has a class of `wp-block-media-text__content`.
+
+As with the group block, a control can set the parent wrapper's pre-selected or
+custom `background-color`. See [color settings](#color-settings).
+
+Controls allow the horizontal and vertical alignment and arrangement of the media
+and text elements. Choices result in the insertion of additional parent wrapper classes
+such as `is-vertically-aligned-center` and `has-media-on-the-right`, as well as either
+an `alignwide` (assumes horizontal padding) or `alignfull` (assumes no horizontal
+padding) class.
+
+Another control toggles the class `is-stacked-on-mobile`.
+
+<Preview>
+  <Story name="core/media-text" height="650px">
+    {`<div class="wp-block-media-text alignwide is-stacked-on-mobile has-media-on-the-right is-vertically-aligned-center has-background has-subtle-background-background-color">
+        <figure class="wp-block-media-text__media">
+          <img src="https://s.w.org/images/core/5.3/Biologia_Centrali-Americana_-_Cantorchilus_semibadius_1902.jpg" alt="" />
+        </figure>
+        <div class="wp-block-media-text__content">
+          <p>The wren<br>Earns his living<br>Noiselessly.</p>
+          <p>— Kobayashi Issa (一茶)</p>
+        </div>
+      </div>`}
+  </Story>
+</Preview>
+
+## Preformatted
+
+The preformatted block allows theming of content within `pre` tags using the `wp-block-preformatted`
+class.
+
+<Preview>
+  {/* eslint-disable no-useless-escape */}
+  <Story name="core/preformatted" height="350px">
+    {`<pre class="wp-block-preformatted">
+       ___________________________
+     < I'm an expert in my field. >
+       ---------------------------
+           \\  ^__^
+            \\ (oo)\\_______
+              (__)\\       )\\\/\\
+                  ||----w |
+                  ||     ||
+      </pre>`}
+  </Story>
+  {/* eslint-enable no-useless-escape */}
+</Preview>
+
+## Pullquote
+
+The pullquote block wraps a `blockquote` element in a `figure` element with the class
+`wp-block-pullquote`.
+
+[Color controls](#color-settings) allow the setting of pre-defined or custom `background-color` and
+text `color`. A styles control adds one of two wrapper classes, `is-style-default` or
```

Comment:
Nit: sometimes this is referred to as `styles control` and sometimes it's referred to as `style controls`. It would be nice to be consistent.

Reply:
I updated for consistency